### ISSUE TITLE
[Accessibility]: Screen reader gets stuck in Home's animating header

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5537]
### Audit type

Screen reader navigation

### User journey audit issue

#5500

### Problem description

A screen reader runs through each item on a page, but on Home (about.sourgraph.com), the reader gets stuck in a loop on our "animating" hero title.
![image](https://user-images.githubusercontent.com/59381432/176556129-fed6cded-c7c7-45dd-908e-c634151a60ff.png)

### Expected behavior

Suggestion: After the 3 title items are read, the screen reader should be able to exit and jump to the next item.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
